### PR TITLE
Feat/add dh favicon

### PR DESCRIPTION
**Pull request by frontend-specialisten**
## Description

Add favicon(s) for DataHub

## References

https://github.com/Energinet-DataHub/geh-aggregations/issues/557

### CODE REVIEW

**Review by MrDach — APPROVED** [review]

File: apps/dh/app-dh/project.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
         "allowedCommonJsDependencies": ["date-fns-tz"],
         "assets": [
           {
-            "glob": "favicon.ico",
-            "input": "libs/dh/shared/assets/src",
+            "glob": "**/*",
+            "input": "libs/dh/shared/assets/src/assets/favicon",
```
Comment (dzhavat) [inline apps/dh/app-dh/project.json:21]: question: Are the favicon files included twice in the final bundle? Once by this entry and once by the next one?

Comment (frontend-specialisten) [reply apps/dh/app-dh/project.json:21]: No what's copied here is a favicon folder which includes the icons in the different sizes

Comment (frontend-specialisten) [reply apps/dh/app-dh/project.json]: Good catch, should be fixed now :)

File: libs/dh/shared/assets/src/favicon/favcicon - Datahub.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+<path d="M30.0159 14.7652C29.9966 14.8365 29.9378 14.9374 29.7614 15.0523C29.5816 15.1694 29.3092 15.2804 28.93 15.37C28.1722 15.5491 27.0927 15.6193 25.7521 15.567C23.0793 15.4628 19.4805 14.8771 15.5966 13.8271C11.7126 12.777 8.30897 11.4696 5.94774 10.2127C4.76349 9.5824 3.86636 8.97781 3.30211 8.44129C3.01973 8.1728 2.84032 7.93963 2.74405 7.74788C2.6496 7.55976 2.64961 7.44299 2.66887 7.37173C2.68814 7.30048 2.74697 7.19961 2.92334 7.08471C3.10311 6.96759 3.37558 6.85659 3.75478 6.76696C4.51251 6.58785 5.59207 6.51767 6.93261 6.56993C9.60546 6.67412 13.2042 7.25985 17.0882 8.30991C20.9721 9.35997 24.3758 10.6674 26.737 11.9242C27.9213 12.5546 28.8184 13.1591 29.3826 13.6957C29.665 13.9642 29.8444 14.1973 29.9407 14.3891C30.0351 14.5772 30.0351 14.694 30.0159 14.7652Z" stroke="#9C9E9F"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M8 8.53639L11.0151 9.36901V8.14637H13.6266C16.2913 8.14637 18.0963 9.29274 19.0415 11.5855L22.6534 12.5829C22.2704 10.7291 21.5702 9.18729 20.5526 7.95759C18.9209 5.98586 16.6375 5 13.7023 5H8V8.53639Z" fill="#05A388"/>
+<circle cx="20" cy="15" r="3" fill="#FED425"/>
+</svg>
```
Comment (dzhavat) [inline libs/dh/shared/assets/src/favicon/favcicon - Datahub.svg:6]: suggestion: rename this file to "Original-DataHub-Favicon.svg" and move it just outside the "assets" folder so we don't include it in the bundle.

File: libs/dh/shared/assets/README.md [hunk 3]
```diff
@@ -1 +1,5 @@
 # DataHub assets
+
+## favicon
```
Comment (dzhavat) [inline libs/dh/shared/assets/README.md:3]:
```suggestion
## Favicon
```

File: libs/dh/shared/assets/README.md [hunk 4]
```diff
@@ -1 +1,5 @@
...
+
+favicons are generated with https://realfavicongenerator.net/ using `favcicon - Datahub.svg`
```
Comment (dzhavat) [inline libs/dh/shared/assets/README.md:5]:
```suggestion
Favicons are generated with https://realfavicongenerator.net/ using `original-datahub-favcicon.svg`
```

**Review by dzhavat — DISMISSED** [review]

**Review by dzhavat — APPROVED** [review]

**Pull request merged**